pytket/extensions/quantinuum/__init__.py

```diff
@@ -17,4 +17,4 @@
 
 # _metadata.py is copied to the folder after installation.
 from ._metadata import __extension_version__, __extension_name__  # type: ignore
-from .backends import QuantinuumBackend, QuantinuumAPI, QuantinuumAPIOffline, Language
+from .backends import QuantinuumBackend, QuantinuumAPI, QuantinuumAPIOffline, Language, get_detection_circuit, remove_leaked_results
```

Comment:
Do we need to make `get_detection_circuit` part of the public API?

Reply:
Nope we don't, will remove.

Reply:
Done